docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -0,0 +1,118 @@
+# CREATE SKILL - NEW COMMAND
+
+`askx new` allows developers to create an `Alexa hosted skill`, and clone the skill project to the local machine, which replaces and optimizes `ask create-hosted-skill` command in CLI V1.
+
+Alexa hosted deployer builds Alexa skills quickly by provisioning required resources automatically. 
+Alexa hosted service will create a new AWS account with a deployment pipeline (CodeBuild), a skill Lambda function, a Git repository (CodeCommit) and an S3 bucket for developers.
+Thus, the developers can then develop on the editor inside the portal or checkout the code using Git and develop with their IDEs.
+
+**STRUCTURE OF INIT COMMAND:**
+
+`askx init [-p | --profile <profile>] [--debug] [-h | --help]`
+
+**OPTIONS DESCRIPTION:**
+
+**profile**: Optional. Specify a profile name to be used. Defaults to use `default` as the profile name, if this option or environmental variable `ASK_DEFAULT_PROFILE` is not set.
+
+**debug**: Optional. Appends a debug message to the standard error.
+
+
+## WORKFLOW:
+
+Users will be asked following questions to create a new skill:
+
+* Prompts user for a `programming language`
+	* Select a programming language.
+* Prompts user for a `deploy delegate` to depoly skill infrastructure
+	* Select a `@ask-cli/hosted-skill-deployer`.
+* Prompts user for a `skill template`
+	* There is one template- Hello World Skill as a default option for now.
+* Prompts user for a `skill name`
+  * Leave this empty to use default skill name or
+  * Enter a value for skill name.
+* Prompts user for a `folder name` for the skill project
+  * Leave this empty to use default folder name or
+  * Enter a value for folder name.
+
+
+# DOWNLOAD SKILL - INIT COMMAND 
+
+`askx init --hosted-skill-id <hosted-skill-id>` -- helps developers migrate an Alexa-Hosted Skill (code + resources) to their local environment.
+
+This command initializes Alexa Hosted Skills by cloning project from the hosted skill service, 
+and provide git-ready environment, so the developers can checkout, pull from, and push to a remote Git repository from their local machines.
+
+**STRUCTURE OF INIT COMMAND:**
+
+`askx init [--hosted-skill-id <hosted-skill-id>] [-p | --profile <profile>] [--debug] [-h | --help]`
+
+**OPTIONS DESCRIPTION:**
+
+**debug**: Optional. Specify a skill-id for the Alexa hosted skill.
+
+**profile**: Optional. Specify a profile name to be used. Defaults to use `default` as the profile name, if this option or environmental variable `ASK_DEFAULT_PROFILE` is not set.
+
+**debug**: Optional. Appends a debug message to the standard error.
+
+
+## WORKFLOW:
+
+Users will be asked the following question to create a new skill:
+
+* Prompts user for a `folder name` for the skill project to store in local machine.
```

Comment:
this was already mentioned above in the workflow section.

Reply:
It's different scenario, but it's not as necessary as the above case, so I will remove this section